From f52b1c6c0ad1e04512ce836701cf2df77539f4d0 Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Fri, 15 Jan 2021 16:18:10 +0800
Subject: [PATCH 27/27] delete pxe and dhcp boot target

This function has not been implemented for the time being.
when the function ready,then would recovery this fucntion back

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 include/configs/iot2050.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/configs/iot2050.h b/include/configs/iot2050.h
index 7c0d340f6b..ec3941b90f 100644
--- a/include/configs/iot2050.h
+++ b/include/configs/iot2050.h
@@ -51,9 +51,7 @@
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0) \
 	func(USB, usb, 1) \
-	func(USB, usb, 2) \
-	func(PXE, pxe, na) \
-	func(DHCP, dhcp, na)
+	func(USB, usb, 2)
 
 #include <config_distro_bootcmd.h>
 
-- 
2.17.1

